src/css/content.scss

```diff
@@ -132,4 +132,10 @@ input:checked + .autoplay-slider:before {
 .betterYT-hidden
 {
   visibility: hidden !important;
+}
+
+ytd-reel-player-overlay-renderer[is-watch-while-mode]>#actions button:hover{
+  background-color: #333333cc;
+  -webkit-transition: .2s;
+  transition: .2s;
```

Comment:
May I know what does this styling do?

Reply:
When the controller is in the video, hovering over it colors it.
Default UI is hard to recognize which button is selected for some videos.